app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -282,9 +282,11 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
 
     private fun showOtherClassifications(otherClassifications: List<Classifier.Classification>) {
         if (otherClassifications.isEmpty()) {
-            footer.visibility = View.GONE
+            otherPredictionsSeparator.visibility = View.GONE
```

Comment:
I see... but if we have the main classification and we have no other classifications, shouldn't we be able to benchmark the main classification as well?

If the main classification is sunflower, for example, I think it would be nice to check how the other services classify it, no?